decidim-core/db/migrate/20170221094835_add_scopes_to_processes.rb

```diff
@@ -0,0 +1,5 @@
+class AddScopesToProcesses < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_participatory_processes, :scope_ids, :integer, array: true, default: []
```

Comment:
LOL, I remember the conversation and I *think* I was against this. I think it's better to don't allow multiple scopes since this opens too many options.